tests/hardhat/util/TokenTestHelpers.ts

```diff
@@ -111,10 +111,10 @@ const deployVTokenDependencies = async <VTokenFactory extends AnyVTokenFactory =
       (await deployVTokenBeacon<VTokenFactory>(
         { kind },
         params.isTimeBased || false,
-        params.blocksPerYear || BSC_BLOCKS_PER_YEAR,
+        params.blocksPerYear === 0 ? 0 : BSC_BLOCKS_PER_YEAR,
```

Comment:
I think we should correct this condition as if we enter blocksPerYear other rather than `BSC_BLOCKS_PER_YEAR` then it will always pick `BSC_BLOCKS_PER_YEAR` not the blocks we entered.

Reply:
Done